resources/js/src/app/components/item/QuantityInput.vue

```diff
@@ -5,7 +5,8 @@
                :value="displayValue"
                @change="setValue($event.target.value)"
                :disabled="waiting"
-               ref="quantityInputField">
+               ref="quantityInputField"
+               :aria-label="$translate('Ceres::Widget.quantityInputLabel')">
```

Comment:
You are using a translation from the `Widget.properties`. This file provides translations to be used in the shop builder in the plentymarkets backend but not for the shop frontend. Because of this, this file isn't loaded in the frontend so the translation might not work here!

Please use a translation key of the `Template.properties` file or add a new one there if required.

Reply:
Thanks @felixdausch,
I wasn't sure about the use of this property file